drivers/display/display_st7735r.c

```diff
@@ -0,0 +1,461 @@
+/*
+ * Copyright (c) 2020 Kim Bøndergaard <kim@fam-boendergaard.dk>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT sitronix_st7735r
+
+#include "display_st7735r.h"
+
+#include <device.h>
+#include <drivers/spi.h>
+#include <drivers/gpio.h>
+#include <sys/byteorder.h>
+#include <drivers/display.h>
+
+#define LOG_LEVEL CONFIG_DISPLAY_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(display_st7735r);
+
+#define ST7735R_CS_PIN		DT_INST_SPI_DEV_CS_GPIOS_PIN(0)
```

Comment:
better retrieve display parameters in a config structure, e.g. https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/display/display_ili9xxx.c#L440

Reply:
:+1: